comp/metadata/inventorychecks/inventorychecksimpl/inventorychecks.go

```diff
@@ -252,7 +276,7 @@ func (ic *inventorychecksImpl) getPayload() marshaler.JSONMarshaler {
 		}
 	}
 
-	return &Payload{
+	return Payload{
```

Comment:
Could you explain the need for this change? 

Previously we declare the function to return an interface `marshaler.JSONMarshaler` now the function return a copncrete type. That type has function defined on its pointer. Then in the new function `getJSONPayload` we get the value and use its pointer. Could we return the pointer version of Payload directly?

Reply:
At the beginning I thought it was not possible to return a pointer through the `expvar.Publish` function, but it works !